david_laster/class4/tunr.rb

```diff
@@ -0,0 +1,97 @@
+##############
+## step one###
+##############
+
+# Give the program access to the necessary gems
+# we know we will need to make HTTP requests to the itunes api, so we will need HTTParty
+# we know we will need to parse json from the response, so we will need json
+require 'httparty'
+require 'json'
+
+
+###############
+## step two ###
+###############
+
+# Fill in the blanks to make this method return songs by the artist specified in term as a hash.
+# remember you will need HTTParty and JSON.parse
+
+# first try and just construct the url with your method and copy it and paste it into the browser to make sure that it works.
+# then use HTTParty to GET the json, and save it to a variable
+# Then use the JSON gem to parse the json into a hash. that will be the return value of this method.
+
+
+def search_itunes(term)
+  base = "https://itunes.apple.com/search?term="
+  url =  base + term
+  response = JSON.parse(HTTParty.get url)
+  return response
```

Comment:
I like that you were explicit here!

alternatively, you could save yourself a line of code here because even if you don't prepend `return` at the beginning of the line - ruby will automatically return the last line before `end`

like so:

```
def search_itunes(term)
  base = "https://itunes.apple.com/search?term="
  url =  base + term
  JSON.parse(HTTParty.get url)
end
```